Client/Client.csproj

```diff
@@ -66,38 +65,26 @@
     <Compile Include="Core\AForge\Video.DirectShow\Internals\ICreateDevEnum.cs" />
     <Compile Include="Core\AForge\Video.DirectShow\Internals\IEnumFilters.cs" />
     <Compile Include="Core\AForge\Video.DirectShow\Internals\IEnumPins.cs" />
-    <Compile Include="Core\AForge\Video.DirectShow\Internals\IFileSourceFilter.cs" />
     <Compile Include="Core\AForge\Video.DirectShow\Internals\IFilterGraph.cs" />
     <Compile Include="Core\AForge\Video.DirectShow\Internals\IFilterGraph2.cs" />
     <Compile Include="Core\AForge\Video.DirectShow\Internals\IGraphBuilder.cs" />
     <Compile Include="Core\AForge\Video.DirectShow\Internals\IMediaControl.cs" />
     <Compile Include="Core\AForge\Video.DirectShow\Internals\IMediaEventEx.cs" />
-    <Compile Include="Core\AForge\Video.DirectShow\Internals\IMediaFilter.cs" />
-    <Compile Include="Core\AForge\Video.DirectShow\Internals\IPersist.cs" />
     <Compile Include="Core\AForge\Video.DirectShow\Internals\IPin.cs" />
     <Compile Include="Core\AForge\Video.DirectShow\Internals\IPropertyBag.cs" />
     <Compile Include="Core\AForge\Video.DirectShow\Internals\IReferenceClock.cs" />
```

Comment:
@DragonzMaster I really don't understand what do you mean. What files should I delete ?